src/main/java/uk/sky/cqlmigrate/CqlPaths.java

```diff
@@ -1,19 +1,24 @@
 package uk.sky.cqlmigrate;
 
+import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableSortedMap;
 
 import java.io.File;
 import java.io.FilenameFilter;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.stream.StreamSupport;
 
 class CqlPaths {
 
     private static final String BOOTSTRAP_CQL = "bootstrap.cql";
-    private static final FilenameFilter CQL_FILE_NAMES = (file, name) -> name.endsWith(".cql");
+    private static final String CQL_FILE_FILTER = "*.cql";
```

Comment:
Good call, yes and this is now addressed.

I tried replacing this with
```java
private static final DirectoryStream.Filter<Path> CQL_FILE_FILTER = path -> path.getFileName().endsWith(".cql");
```
but I got this exception in the functional tests:
>[main] INFO uk.sky.cqlmigrate.CqlMigratorImpl - Loading cql files from [cql_bootstrap]
[main] INFO uk.sky.cqlmigrate.KeyspaceBootstrapper - Keyspace not found, applying null
java.lang.NullPointerException
	at uk.sky.cqlmigrate.CqlLoader.load(CqlLoader.java:36)
	at uk.sky.cqlmigrate.KeyspaceBootstrapper.lambda$bootstrap$8(KeyspaceBootstrapper.java:27)
	at uk.sky.cqlmigrate.CqlPaths.applyBootstrap(CqlPaths.java:53)
	at uk.sky.cqlmigrate.KeyspaceBootstrapper.bootstrap(KeyspaceBootstrapper.java:25)
	at uk.sky.cqlmigrate.CqlMigratorImpl.migrate(CqlMigratorImpl.java:88)
	at uk.sky.cqlmigrate.CqlMigratorImpl.migrate(CqlMigratorImpl.java:65)
	at uk.sky.cqlmigrate.example.CqlMigrateInvoker.doMigrate(CqlMigrateInvoker.java:40)
	at uk.sky.cqlmigrate.example.CmdLineEntryPoint.main(CmdLineEntryPoint.java:17)

If it's OK with you, I'd rather get the working version in and consider looking at this as an optimisation later, since this is a blocking issue for us.